examples/src/pages/Migration.tsx

```diff
@@ -71,7 +71,13 @@ export function Migration() {
       async function addModel(options: AddModelOptions) {
         try {
           const model = await viewer.addModel(options);
-          viewer.fitCameraToModel(model);
+          const cameraConfig = model.getCameraConfiguration();
+          if (cameraConfig) {
```

Comment:
You don't use your new method here by the way.

Reply:
Thx, forgot to update.